public/js/actions/AtomActions/getSuggestionsForLatestContent.js

```diff
@@ -34,44 +34,32 @@ function errorReceivingSuggestionsForLatestContent(error) {
 const distinct = array => array.filter((value, index, self) => self.indexOf(value) === index);
 const flatten = array => [].concat.apply([], array);
 
-//Returns set of all tags in the given content
-function getTags(contentArray) {
-  return distinct(flatten(
-    contentArray.map(content => content.tags.filter(tag => tag.type === "keyword").map(tag => tag.id))
-  ));
-}
-
-//Returns a map of tagId to suggestions from the targeting api, filtering out non-atoms
-function getTagToTargetUrls(tags) {
-  const tagToTargetAtoms = {};
+//Returns an array of target atom urls for the given tags
+function getTargetUrls(tags) {
+  const tagIds = tags.filter(tag => tag.type === "keyword").map(tag => tag.id);
 
-  const fetch = (tag) => {
-    return fetchTargetsForTag(tag)
-      .then(targets => {
-        const filtered = targets.filter(target => target.url.includes("/atom/"));
-        const urls = filtered.map(target => target.url);
-        if (filtered.length > 0) {
-          tagToTargetAtoms[tag] = urls;
-        }
-        return Promise.resolve();
-      });
-  };
+  return fetchTargetsForTags(tagIds).then(targets => {
+    const filtered = targets.filter(target => target.url.includes("/atom/"));
+    return filtered.map(target => target.url);
+  });
+}
 
-  //Sequentially fetch targets for each tag
-  return tags.reduce((p, tag) => p.then(() => fetch(tag)), Promise.resolve())
-    .then(() => Promise.resolve(tagToTargetAtoms));
+const AllSnippets = ["guide","qanda","timeline","profile"];
+function isSnippet(atomType) {
```

Comment:
This kind of code is all over the place. I wonder if adding an `isSnippet` flag to all atoms would make our lives easier.

Reply:
you mean in the model? It could be useful, though we'd have to apply it to all existing snippets

Reply:
Yeah it's just a thought, but since snippet seem to be part of our everyday vocabulary, and we might grow the family in the future, why not? I wonder what @mchv thinks about that. That flag would need to be added to `Atom` in order to be really effective.

Reply:
I would not add that to the model but in the client if this is useful with an extension method on the `Atom` class

Reply:
i.e 
```scala
implicit class RichAtom(val atom: Atom) extends AnyVal {
   def isSnippet =atom.type == AtomType.Qanda || atom.type == AtomType.Profile || atom.type == AtomType.Guideline ||atom.type == AtomType.Timeline
```

Reply:
Yes, that makes perfect sense.